src/main/java/com/desiato/puresynth/services/AuthenticationService.java

```diff
@@ -26,13 +26,34 @@ public class AuthenticationService {
     public PureSynthToken authenticate(AuthenticationRequestDTO request) {
         log.info("Attempting to authenticate user with email: {}", request.email());
 
-        User user = findUserOrThrow(request);
+        User user;
+        try {
```

Comment:
What's point of this?

Reply:
Since I was not getting where the problem was, I used try catch blocks to see where the error was coming from

Reply:
I know you used it for debugging. But why do you want to merge it? It make code longer, harder to read and doesn't bring any new feature or increase raliability.

Reply:
yes, I was not thinking about their uselessness after the issue was solved.